assets/src/edit-story/elements/media/display.js

```diff
@@ -31,7 +31,8 @@ import getMediaSizePositionProps from './getMediaSizePositionProps';
 
 const Element = styled.div`
   ${elementFillContent}
-  ${({ showPlaceholder }) => showPlaceholder && `background-color: #C4C4C4;`}
+  ${({ showPlaceholder }) =>
+    showPlaceholder && `background-color: #C4C4C4;`}
```

Comment:
Hm, seems like you have a bad version of eslint or wrong config here?

Reply:
Hmm, the version seems to be correct, haven't changed the config either. Odd, will need to investigate.